core/src/main/java/de/danoeh/antennapod/core/feed/LocalFeedUpdater.java

```diff
@@ -70,7 +70,21 @@ private static void tryUpdateFeed(Feed feed, Context context) throws IOException
         Set<String> mediaFileNames = new HashSet<>();
         for (DocumentFile file : documentFolder.listFiles()) {
             String mime = file.getType();
-            if (mime != null && (mime.startsWith("audio/") || mime.startsWith("video/"))) {
+            if (mime == null) {
+                continue;
+            }
+
+            MediaType mediaType = MediaType.fromMimeType(mime);
+            if (mediaType == MediaType.UNKNOWN) {
+                String path = file.getUri().toString();
+                int i = path.lastIndexOf('.');
```

Comment:
Could you please rename this to something like `fileExtensionPosition`? Would be more clear than just `i`, which looks like a loop variable

Reply:
Thanks for the review! I have addressed both points in the last commit =)